unlock-app/src/components/interface/LogIn.tsx

```diff
@@ -0,0 +1,151 @@
+// eslint-disable-next-line no-unused-vars
+import React, { FormEvent } from 'react'
+import styled from 'styled-components'
+import { connect } from 'react-redux'
+// eslint-disable-next-line no-unused-vars
```

Comment:
this should not be necessary - the fix is to copy the config inside paywall to the top-level eslintrc:

```js
  rules: {
    'no-unused-vars': 'off',
    '@typescript-eslint/no-unused-vars': [
      'error',
      {
        vars: 'all',
        args: 'after-used',
        ignoreRestSiblings: true,
      },
    ],
```

Obviously, I am suggesting a quick follow-up PR, not changing this one

Reply:
Do these get turned off for everything? The issue is that eslint can't seem to correctly handle these in TS files, but we do want the coverage everywhere else.

Reply:
`@typescript-eslint/no-unused-vars` catches unused variables, so we get the best of both worlds

Reply:
I mean, the typescript compiler also catches unused variables thanks to our TSConfig.